importer: remove 'add-cookies' listener -- now handled in muon

bpowers opened the pull request:
## Requires
- [ ] https://github.com/brave/muon/pull/365 to be merged
- [ ] Muon reference for browser-laptop needs to be updated

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

## Reviewer Checklist:

Tests


- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

Fixes brave/muon#320

darkdh commented on app/importer.js:
```diff
@@ -233,30 +232,6 @@ const shouldSkipCookie = (cookie) => {
 }
 module.exports.shouldSkipCookie = shouldSkipCookie
 
-importer.on('add-cookies', (e, cookies) => {
```
as mentioned in https://github.com/brave/muon/issues/320#issuecomment-340082014. we should also remove shouldSkipCookie and its unittest

bpowers commented:
@darkdh I updated both PRs -- the browser-laptop one now removes `shouldSkipCookie` and its unit test, and when I rebased that one I noticed I had missed a URL to check in the C++ version.

bsclifton commented:
Closing as stale
